packages/fxa-content-server/server/lib/routes/get-index.js

```diff
@@ -66,6 +67,7 @@ module.exports = function(config) {
     sentryDsn: SENTRY_CLIENT_DSN,
     staticResourceUrl: STATIC_RESOURCE_URL,
     subscriptions: SUBSCRIPTIONS,
+    surveys: SURVEYS,
```

Comment:
There are config related tests in the `get-index.js` test suite.  Let's add to that.